modules/tinymce/src/core/test/ts/browser/EditorManagerTest.ts

```diff
@@ -107,6 +108,9 @@ describe('browser.tinymce.core.EditorManagerTest', () => {
           // Destroy the editor by setting innerHTML common ajax pattern
           viewBlock.update('<textarea id="' + editor1.id + '"></textarea>');
 
+          // We need to remove the sink since it's added to the body
+          Selectors.one('.tox-silver-sink').each(Remove.remove);
```

Comment:
Yes, if you look at line 109 it removes the editor by setting the innerHTML of the viewblock to a textarea and re-initializes on that. But since the sink is added to the body not the viewBlock element it will linger there when the editor is removed. It will be very hard for us I think to remove the sink in this case. Better to change the way the sink gets added so that it gets added before/after the target element then this case would work and also the issues with scrolling view ports that some users have reported. The sink should really not be in the body not sure if there is a ticket on changing that but that change is out of scope of this PR this is more to just get all tests passing if say the chunking is removed that is how I found this. The shadow dom test checks for a sink in the body and it will find one since this doesn't clean up the sink.